app/src/main/java/com/tr4n/moviedb/ui/detail/MovieDetailViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package com.tr4n.moviedb.ui.detail
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.viewModelScope
+import com.tr4n.moviedb.base.BaseViewModel
+import com.tr4n.moviedb.data.model.MovieDetail
+import com.tr4n.moviedb.data.source.MoviesRepository
+import kotlinx.coroutines.launch
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class MovieDetailViewModel : BaseViewModel(), KoinComponent {
+    private val moviesRepository : MoviesRepository by inject()
+    val movieDetail = MutableLiveData<MovieDetail>()
+    fun getMovieDetails(movieId : Long) {
```

Comment:
viết tách ra 1 dòng phân cách 
```suggestion
    val movieDetail = MutableLiveData<MovieDetail>()
    
    fun getMovieDetails(movieId : Long) {
```